resources/config.py

```diff
@@ -143,19 +160,22 @@ def get(self, identifier):
                 description: Config information and contents
                 content:
                   application/json:
-                    schema: ConfigShowSchema
+                    schema: ConfigItemResponseSchema
             404:
                 description: |
                     When config doesn't exist, object is not a config or user doesn't have access to this object.
         """
-        return super(ConfigResource, self).get(identifier)
+        return super().get(identifier)
 
-    def create_object(self, obj):
+    def _create_object(self, spec, parent, share_with, metakeys):
         try:
             return Config.get_or_create(
-                obj.data["cfg"],
-                obj.data["family"],
-                obj.data.get("config_type", "static")
+                spec.data["cfg"],
+                spec.data["family"],
+                spec.data.get("config_type", "static"),
```

Comment:
Is there a reason for not handling this default in the schema?

Reply:
I've checked and `static` is set on missing, so do we still have to handle it here?

Reply:
:+1: